spec/swagger.yaml

```diff
@@ -2389,11 +2410,11 @@ definitions:
                 example: client_credentials
               client_id:
                 type: string
-                description: Contact Twine to get a client id and secret.
+                description: Contact Fitbit Plus to get a client id and secret.
```

Comment:
'Contact Fitbit Plus' seems incomplete, may be 'Contact Fitbit Plus administrator' or similar.